static/css/components/mybooks-details.less

```diff
@@ -17,8 +17,7 @@
 }
 
 .breadcrumb-wrapper {
-  display: flex;
-  flex: 1;
+  display: block;
```

Comment:
We may be able to `flex-wrap: wrap;` on `breadcrumb-wrapper` which may be a better solution than `display: block`